oat-sa/tao-core
# open eslint task to multiple report format

Enable multiple reports format for the ESLint task  in TAO (those formats are expected for the CI)

How to test this pull request :  
```
cd views/build
npm run lint -- --extension=taoItems --format=checkstyle
npm run lint -- --extension=taoItems --format=json
```
This should have created the following reports : 
```
./views/build/reports/CHECKSTYLE-taoItems.xml
./views/build/reports/JSON-taoItems.json
```

The test reports have also been improved in order to contain meaningfull suite and test names (based on namespaces)
to test them :  
```
cd views/build
npx grunt connect:test qunit_junit taoitemstest
```
Check the reports `./views/build/reports/TEST-*`
